02.calendar/cal.rb

```diff
@@ -0,0 +1,52 @@
+#!/Users/heureux/.rbenv/versions/2.6.5/bin/ruby
+
+require 'optparse'
+require 'date'
+
+days = ['日','月','火','水','木','金','土']
+
+opt = OptionParser.new
+opt.on('-y')
+opt.on('-m')
+opt.parse!(ARGV)
+
+if ARGV.empty?
+  targetYear = Date.today.year.to_i
+  targetMonth = Date.today.month.to_i
+else
+  arguments = ARGV
+  targetYear = arguments[0].to_i
+  targetMonth = arguments[1].to_i
```

Comment:
```suggestion
  targetYear = ARGV[0].to_i
  targetMonth = ARGV[1].to_i
```

`arguments = ARGV`としても変数名によって新しい情報が付与されていないので、↑のようなコードで十分だと思います〜。